### PR TITLE
Bug/30637 Pages availability issues

### PR DESCRIPTION
**Pull request by GeorgeChatzigiannis**
- [x] Include opening hours for generate pins availability

### CODE REVIEW

File: admin/services/business-availability.service.js [hunk 1]
```diff
@@ -113,6 +113,7 @@ businessAvailabilityService.determineAccessArrangementsEligibility = (checkWindo
  */
 businessAvailabilityService.getAvailabilityData = async (dfeNumber, checkWindowData) => {
   const currentDate = moment.utc()
+  const isWithinOpeningHours = currentDate.hour() >= 8 && currentDate.hour() < 16
```
Comment (jon-shipley) [inline admin/services/business-availability.service.js]: @danwilliams - how will this play with your mod change?

Comment (GuyHarwood) [reply admin/services/business-availability.service.js]: not very well given that we are not taking into account the timezone of the school.  the offset is available as part of the session object

**Comment by GeorgeChatzigiannis** [comment]
@GuyHarwood this is now using `moment-timezone` to determine the correct date time

**Comment by activemq** [comment]
@GeorgeChatzigiannis - Additional pages found that need to be covered - 
http://localhost:3001/pupil-register/pupil/add
http://localhost:3001/pupil-register/pupil/add-batch-pupils
http://localhost:3001/access-arrangements/select-access-arrangements?
http://localhost:3001/attendance/confirm-and-submit

**Comment by GeorgeChatzigiannis** [comment]
@activemq I don't think there is a limitation for pupil register. The page should always be visible as far as I know

**Pull request merged**